spec/spec_helper.rb

```diff
@@ -24,12 +22,24 @@ class SecondUser < Minidoc
   attribute :age, Integer
 end
 
+class ValidationsUser < User
```

Comment:
ah, I initially moved this here because I thought I was going to use it in multiple specs, but then I ended up not doing that. Can move it back